lessthan3/maestro-react-player
# Hard code autoplay to true for Twitch SDK happiness

Motivation for this PR:
Recently it has surfaced that the player will not rerender when a new Twitch URL is passed. This means, when you change the URL of your live video, it will not appear until after a refresh, or play/pause, or switching channels. More details on this bug here https://github.com/CookPete/react-player/issues/498

Yes, this is hard coding, yes it looks hacky, but it's actually not! The `playing` prop should be used to play and pause the video, not to control the `autoplay` value for loading the Twitch SDK. In our case, we want Twitch videos to always autoplay (I think).

Testing instructions:
create two twitch_live type videos that are both currently live
put them in your livestreams collection
see the top video play
switch the order, see the new video play
switch back and forth a few times -- should work indefinitely

Has been tested on Chrome, Firebox, and iOS.
